src/content/docs/crowdin/translation-process/crowdin-ai.mdx

```diff
@@ -44,8 +44,8 @@ To configure AI prompts, follow these steps:
   2. In the **Prompts** tab, click **Add prompt**. Alternatively, you can go ahead with creating prompt samples clicking **Try a sample**. Crowdin allows you to quickly set up pre-configured prompts that are ready to perform effectively from the moment they are created. While these samples are designed to work good initially, you can customize them to your personal preferences.
   3. In the appeared dialog, set the prompt parameters:
       * Select **Enabled**
-      * Select the prompt type (i.e., Pre-translate, AI in editor, or Create custom type)
-      * Custom type name (Specific to the custom type prompt)
+      * Select the prompt type (i.e., Pre-translate, AI in editor, [QA check](#ai-qa-check), or Create custom type)
```

Comment:
It would be great to add links for the Pre-translate and AI in editor prompt types (they are present in the crowdin enterprise article)